$NetBSD: patch-CMakeLists.txt,v 1.6 2022/07/27 08:40:58 adam Exp $

Do not look for X11 on macOS.
We don't have readline as a module.
Do not bother with uninstaller.

--- CMakeLists.txt.orig	2022-07-13 16:10:31.000000000 +0000
+++ CMakeLists.txt
@@ -76,7 +76,9 @@ find_package(ICU REQUIRED COMPONENTS dat
 # no, gettext executables are not required when NLS is deactivated
 find_package(Gettext)
 
+if(NOT APPLE)
 find_package(X11)
+endif(NOT APPLE)
 
 if(NOT WIN32)
 	# needed to get some SDL2 defines in... (as of rev31694 -D_GNU_SOURCE=1 is required!)
@@ -500,7 +502,6 @@ if(ENABLE_GAME OR ENABLE_TESTS)
 # TODO: can use find_package and Fontconfig::Fontconfig with cmake 3.14+
 	pkg_check_modules(FONTCONFIG REQUIRED fontconfig>=2.4.1)
 	pkg_check_modules(PANGO REQUIRED pango>=1.22.0)
-	pkg_check_modules(LIBREADLINE readline)
 endif()
 
 if(ENABLE_TESTS)
@@ -583,11 +584,6 @@ endif()
 # uninstall
 #
 
-configure_file(
-	"${CMAKE_SOURCE_DIR}/cmake/uninstall.cmake.in"
-	"${CMAKE_CURRENT_BINARY_DIR}/uninstall.cmake"
-	IMMEDIATE @ONLY
-)
 
 add_custom_target(uninstall
 	"${CMAKE_COMMAND}" -P "${CMAKE_CURRENT_BINARY_DIR}/uninstall.cmake"
